### ISSUE TITLE
Can the purchase method in BillingProcessor support Fragment? 

### ISSUE THREAD

**Issue by memorycj** [anjlab/android-inapp-billing-v3#383]
After the startIntentSenderForResult is started by the activity, the Fragment cannot receive the callback of onActiviyResult.

lib version  : 1.0.40

Thanks open source

**Comment by edmundsiah81** [comment]
Yes you can, you just need to know which activity is running for your fragment. Then on that inactivityresult, include super.onActivityResult(requestCode, resultCode, data);
